itest/godot/ManualFfiTests.gd

```diff
@@ -35,7 +35,16 @@ func test_export():
 	var node = Node.new()
 	obj.object_val = node
 	assert_eq(obj.object_val, node)
-
+	
+	var texture_val_meta = obj.get_property_list().filter(
+		func(el)->bool:
+			return el["name"] == "texture_val"
+			).front()
```

Comment:
```suggestion
	var texture_val_meta = obj.get_property_list().filter(
		func(e): return e["name"] == "texture_val"
	).front()
```

Reply:
The suggestion was reformatting and stripping of the return value, why did you mark it as resolved?

Reply:
I don't know, sry.

Reply:
All good, no worries! I don't want to spend too much time on these little issues 🙂